supported.json

```diff
@@ -29,7 +29,7 @@
   },
   "httpx": {
     "versions": ["0.0.1", "0.0.2", "0.0.3", "0.0.4", "0.0.5", "0.0.6", "0.1.0"],
-    "docs-url": "https://k6.io/docs/javascript-api/jslib/httpx"
+    "docs-url": "https://grafana.com/docs/k6/latest/javascript-api/jslib/httpx"
   },
   "expect": {
     "versions": ["0.0.4", "0.0.5"],
```

Comment:
This librarary has a lin, but it actually just redirects to k6chjaijs docs. I guess it is the expect library from there ... extracted?

But this redirect isn't happening in the new docs and ... ot be honest is not very obvious that this is what it is to begin with.

I also didn't change k6chaijs to link to it's docs - it still links to its github, which I take it is on purpose? 

cc @sniku @oleiade as the last people touching this.